strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/TrashController.java

```diff
@@ -22,234 +16,204 @@
     }
 
     @ApiOperation(value = "Used to delete the trash for all repositories.",
-                  position = 2)
-    @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = "The trash for all repositories was successfully removed."),
-                            @ApiResponse(code = 500,
-                                         message = "An error occurred!") })
+            position = 2)
+    @ApiResponses(value = {@ApiResponse(code = 200,
+            message = "The trash for all repositories was successfully removed."),
+            @ApiResponse(code = 500,
+                    message = "An error occurred!")})
     @PreAuthorize("hasAuthority('MANAGEMENT_DELETE_ALL_TRASHES')")
     @RequestMapping(method = RequestMethod.DELETE,
-                    produces = MediaType.TEXT_PLAIN_VALUE)
+            produces = MediaType.TEXT_PLAIN_VALUE)
     public ResponseEntity delete()
-            throws IOException
-    {
-        try
-        {
+            throws IOException {
+        try {
             repositoryManagementService.deleteTrash();
 
             logger.debug("Deleted trash for all repositories.");
-        }
-        catch (ArtifactStorageException e)
-        {
+        } catch (ArtifactStorageException e) {
             logger.error(e.getMessage(), e);
 
             return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
-                                 .body(e.getMessage());
+                    .body(e.getMessage());
         }
 
         return ResponseEntity.ok("The trash for all repositories was successfully removed.");
     }
 
     @ApiOperation(value = "Used to undelete the trash for a path under a specified repository.",
-                  position = 3)
-    @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = "The trash for '${storageId}:${repositoryId}' was restored successfully."),
-                            @ApiResponse(code = 400,
-                                         message = "An error occurred!"),
-                            @ApiResponse(code = 404,
-                                         message = "The specified (storageId/repositoryId/path) does not exist!") })
+            position = 3)
+    @ApiResponses(value = {@ApiResponse(code = 200,
+            message = "The trash for '${storageId}:${repositoryId}' was restored successfully."),
+            @ApiResponse(code = 400,
+                    message = "An error occurred!"),
+            @ApiResponse(code = 404,
+                    message = "The specified (storageId/repositoryId/path) does not exist!")})
     @PreAuthorize("hasAuthority('MANAGEMENT_UNDELETE_TRASH')")
     @RequestMapping(value = "{storageId}/{repositoryId}/{path:.+}",
-                    method = RequestMethod.POST,
-                    produces = MediaType.TEXT_PLAIN_VALUE)
+            method = RequestMethod.POST,
+            produces = MediaType.ALL_VALUE)
```

Comment:
Why have you changed this to `MediaType.ALL_VALUE`.

Reply:
Some tests did not pass. Apparently, this is caused by the update Spring Framework. I have not yet found a solution to the problem.